frontend/templates/static/js/api_calls.js

```diff
@@ -0,0 +1,5 @@
+const baseUrl = 'http://127.0.0.1:8000/api/v1/search/?format=json'
+function searchFlight(from_location, to_location, depature_time) {
```

Comment:
Looks like this function is yet to be implemented. If that's the case, remove it for this PR.